source/scss/_pf-component-alert.scss

```diff
@@ -0,0 +1,38 @@
+//
+// Alerts
+// --------------------------------------------------
+
+.alert {
+  padding-left: $spacer;
```

Comment:
Can you change the variables `$alert-padding-x` and `$alert-padding-y` instead of doing it here?